pkg/util/iptables/iptables_test.go

```diff
@@ -1198,3 +1198,114 @@ func TestRestoreAllWaitBackportedIptablesRestore(t *testing.T) {
 		t.Errorf("expected failure")
 	}
 }
+
+// TestExtractLines tests that
+func TestExtractLines(t *testing.T) {
+	type args struct {
+		count int
+		line  int
+		lines []byte
+	}
+	lines := "Line0: 0\nLine1: 1\nLine2: 2\nLine3: 3\nLine4: 4\nLine5: 5\nLine6: 6\nLine7: 7\nLine8: 8\nLine9\nLine10: 10"
+	tests := []struct {
+		name string
+		args args
+		want []LineData
+	}{
+		{
```

Comment:
what about adding some edge cases like?
All zeros: count 0 - line 0
Out of bounds: count 11 - line 10
...

Reply:
Good advice, got

Reply:
for `Out of bounds: count 11 - line 10`: maybe we also have this case, when count > line,  startIndex starts from line 0